src/acceleration/BaseQNAcceleration.hpp

```diff
@@ -161,6 +161,9 @@ class BaseQNAcceleration : public Acceleration {
   /// Data IDs of data not involved in IQN coefficient computation.
   std::vector<int> _secondaryDataIDs;
 
+  /// Data IDs of all coupling data.
+  std::vector<int> _cplDataIDs;
+
```

Comment:
Is `_dataIDs` then only primary data?
If yes, I would suggest to rename: `_dataIDs` -> `_primaryDataIDs` and `cplDataIDs` -> `_dataIDs`.